pdns/pdns_recursor.cc

```diff
@@ -3967,6 +3967,12 @@ static int serviceMain(int argc, char*argv[])
 
   SyncRes::s_qnameminimization = ::arg().mustDo("qname-minimization");
 
+  if (SyncRes::s_qnameminimization) {
+    // With an empty cache, a rev ipv6 query with dnssec enabled takes
+    // almost 100 queries. Default maxqperq is 60.
+    SyncRes::s_maxqperq = SyncRes::s_maxqperq * 5 / 3;
```

Comment:
I find this arithmetic a bit confusing, especially if the value of `max-qperq` has been updated from the default. Perhaps we could set it to `std::max(SyncRes::s_maxqperq, 100)` when qname minimization has been enabled instead?